### ISSUE TITLE
Bug: cannot create or drop database with custom config

### ISSUE THREAD

**Issue by masb0ymas** [tada5hi/typeorm-extension#267]
i cannot create or drop my database with custom configuration provided in your documentation

this is my code :
```ts
import { DataSourceOptions } from 'typeorm'
import { dropDatabase } from 'typeorm-extension'

void (async () => {
  const dataSourceOptions: DataSourceOptions = {
    type: 'postgres',
    host: 'localhost',
    username: 'masb0ymas',
    password: 'mypassword',
    database: 'express-typeorm',
    port: 5432,
  }

  // Create the database with specification of the DataSource options
  await dropDatabase({ options: dataSourceOptions })
})()
```

run script : 
```sh
npx ts-node ./src/database/dropDB.ts
```

the error message says: `error: database "masb0ymas" does not exist`

Please help me, find solution with the problem.

### Versions
- Node: 14.18.1
- OS: macOS 11.6

### Reproduction

<details open>
<summary>Additional Details</summary>
</details>

**Comment by tada5hi** [comment]
The problem here is, that postgres requires an `initialDatabase` to connect to. 
By default the name is equal to the name of the database user (in your case: masb0ymas)

**Comment by masb0ymas** [comment]
the `createDatabase` function is working normally, but the `dropDatabase` threw error `error: database "masb0ymas" does not exist`.

is `initialDatabase` parameter exist in `dropDatabase` function ?

```sh
dropDatabase({ options: dataSourceOptions, initialDatabase: '<db-name>' })
```

**Comment by tada5hi** [comment]
Actually no 🙈. Good catch! I will fix this today and publish a new release.

**Comment by tada5hi** [comment]
Sorry, did not made it in time yesterday.
Issue should be fixed in e571418bf6d1598aae92e02d1b3409ad20f45328

**Comment by masb0ymas** [comment]
Thanks 👍
